workflows/src/main/kotlin/com/r3/corda/lib/ci/SyncKeyMappingFlow.kt

```diff
@@ -48,6 +65,8 @@ class SyncKeyMappingFlow(private val session: FlowSession, private val tx: WireT
     }
 
     private fun extractConfidentialIdentities(): List<AbstractParty> {
+        tx
```

Comment:
even better make function take as a parameter transaction, then you don't need to do additional nullability check (you already did it in call site)